### PR TITLE
improve: updated misc styling of input elements

### PR DESCRIPTION
**Pull request by luanvdw**

### CODE REVIEW

File: src/components/combo-box/ComboBox.tsx [hunk 1]
```diff
@@ -176,8 +176,8 @@ function ComboBoxContainer<OptionKey extends Key = string>({
             ref={containerRef}
             className={cn(
               "flex items-center w-full relative space-x-2",
-              "rounded-md shadow-sm border border-gray-300 dark:border-gray-700",
```
Comment (madebysid) [inline src/components/combo-box/ComboBox.tsx:179]: No shadow anymore?

File: src/components/combo-box/ComboBox.tsx [hunk 2]
```diff
@@ -176,8 +176,8 @@ function ComboBoxContainer<OptionKey extends Key = string>({
...
-              "px-3 py-1.5",
+              "rounded border border-gray-300 dark:border-gray-700",
+              "px-3 py-2.5",
```
Comment (madebysid) [inline src/components/combo-box/ComboBox.tsx:180]: Let's just make it p-3, these two are so similar

File: src/components/select/Select.tsx [hunk 3]
```diff
@@ -143,8 +143,8 @@ function SelectContainer<OptionKey extends Key = string>({
             {...mergeProps(buttonProps, focusProps)}
             className={cn(
               "inline-flex w-full items-center",
-              "rounded-md shadow-sm border border-gray-300 dark:border-gray-700",
-              "px-3 py-1.5",
+              "rounded border border-gray-400 dark:border-gray-700",
+              "px-3 py-2.5",
```
Comment (madebysid) [inline src/components/select/Select.tsx:147]: Let's make this p-3 as well

File: src/components/combo-box/ComboBox.tsx (hunk 1 above)
Comment (luanvdw) [reply src/components/combo-box/ComboBox.tsx:179]: Nope, got removed from the latest designs.

**Pull request merged**